{
  "type": "issue",
  "issue": {
    "id": 3245170731,
    "node_id": "I_kwDOABII587BbWAr",
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/33015",
    "repository_url": "https://api.github.com/repos/bitcoin/bitcoin",
    "labels_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/33015/labels%7B/name%7D",
    "comments_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/33015/comments",
    "events_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/33015/events",
    "html_url": "https://github.com/bitcoin/bitcoin/issues/33015",
    "number": 33015,
    "state": "open",
    "state_reason": null,
    "title": "intermittent timeout in wallet_signer.py : sendall timed out",
    "body": "https://cirrus-ci.com/task/6466141373071360?logs=ci#L1714:\n```bash\n[17:24:42.699]  node1 2025-07-18T21:24:35.153536Z [scheduler] [net.cpp:2387] [void CConnman::DumpAddresses()] [net] Flushed 0 addresses to peers.dat  0ms \n[17:24:42.699]  node0 2025-07-18T21:24:35.182594Z [scheduler] [net.cpp:2387] [void CConnman::DumpAddresses()] [net] Flushed 0 addresses to peers.dat  1ms \n[17:24:42.699]  test  2025-07-18T21:24:41.997879Z TestFramework (ERROR): Called Process failed with 'error: timeout on transient error: Could not connect to the server 127.0.0.1:21617 (error code 0 - \"timeout reached\") \n[17:24:42.699]                                    Make sure the bitcoind server is running and that you are connecting to the correct RPC port.\n[17:24:42.699]                                    Use \"bitcoin-cli -help\" for more info.\n[17:24:42.699]                                    '\n[17:24:42.699]                                    Traceback (most recent call last):\n[17:24:42.699]                                      File \"/ci_container_base/test/functional/test_framework/test_framework.py\", line 195, in main\n[17:24:42.699]                                        self.run_test()\n[17:24:42.699]                                      File \"/ci_container_base/ci/scratch/build-i686-pc-linux-gnu/test/functional/wallet_signer.py\", line 58, in run_test\n[17:24:42.699]                                        self.test_valid_signer()\n[17:24:42.699]                                      File \"/ci_container_base/ci/scratch/build-i686-pc-linux-gnu/test/functional/wallet_signer.py\", line 212, in test_valid_signer\n[17:24:42.699]                                        res = hww.sendall(recipients=[{dest:0.5}, hww.getrawchangeaddress()], add_to_wallet=False)\n[17:24:42.699]                                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n[17:24:42.699]                                      File \"/ci_container_base/test/functional/test_framework/test_node.py\", line 873, in __call__\n[17:24:42.699]                                        return self.cli.send_cli(self.command, *args, **kwargs)\n[17:24:42.699]                                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n[17:24:42.699]                                      File \"/ci_container_base/test/functional/test_framework/test_node.py\", line 950, in send_cli\n[17:24:42.699]                                        raise subprocess.CalledProcessError(returncode, p_args, output=cli_stderr)\n[17:24:42.699]                                    subprocess.CalledProcessError: Command '['bitcoin', '-m', 'rpc', '-nonamed', '-datadir=/ci_container_base/ci/scratch/test_runner/test_runner_‚Çø_üèÉ_20250718_205643/wallet_signer_218/node1', '-rpcwallet=hww', '-named', 'sendall', 'recipients=[{\"bcrt1qw74qtc8277thn0y0v8qd5xahn583utnftrn6f5\": 0.5}, \"bcrt1q9klcdx4zrgf7m95zhpwfm0lalk92a2rh3swq85\"]', 'add_to_wallet=false']' returned non-zero exit status 1.\n[17:24:42.699]  test  2025-07-18T21:24:42.002020Z TestFramework (DEBUG): Closing down network thread \n```",
    "user": {
      "login": "fanquake",
      "id": 863730,
      "node_id": "MDQ6VXNlcjg2MzczMA==",
      "avatar_url": "https://avatars.githubusercontent.com/u/863730?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/fanquake",
      "html_url": "https://github.com/fanquake",
      "followers_url": "https://api.github.com/users/fanquake/followers",
      "following_url": "https://api.github.com/users/fanquake/following%7B/other_user%7D",
      "gists_url": "https://api.github.com/users/fanquake/gists%7B/gist_id%7D",
      "starred_url": "https://api.github.com/users/fanquake/starred%7B/owner%7D%7B/repo%7D",
      "subscriptions_url": "https://api.github.com/users/fanquake/subscriptions",
      "organizations_url": "https://api.github.com/users/fanquake/orgs",
      "repos_url": "https://api.github.com/users/fanquake/repos",
      "events_url": "https://api.github.com/users/fanquake/events%7B/privacy%7D",
      "received_events_url": "https://api.github.com/users/fanquake/received_events",
      "type": "User",
      "site_admin": false,
      "patch_url": null
    },
    "labels": [],
    "assignees": [],
    "author_association": "MEMBER",
    "locked": false,
    "comments": 0,
    "created_at": "2025-07-19T10:02:07Z",
    "updated_at": "2025-07-19T10:02:07Z"
  },
  "events": []
}